README.md

```diff
@@ -47,10 +45,7 @@ An example of a denied admission would be:
 
 An example of no policy matched:
 1. If the image does not match against any policy
-1. Fallback to [deprecated policy-controller validation behavior](#deprecated-policy-controller-validation-behavior)
-1. Validation will be attempted against the secret defined under `cosign.secretKeyRef.name` during helm installation.
-  1. If a valid signature or attestation is obtained, image is admitted
-  1. If no valid signature or attestation is obtained, image is denied
+1. The image is admitted
```

Comment:
Maybe simplify these three lines to just say:
```
**If an image does not match a policy, it will be admitted**
```

Also, I think we should add a flag / env variable or some other configuration option. We might want to add another ConfigMap anyways (for adding upcoming multi TUF root work), so that might be something. I think having an option that says if an image does NOT match any policy should be denied should be there.
Maybe create an issue for it?

Reply:
> Maybe simplify these three lines to just say: **If an image does not match a policy, it will be admitted**

Sounds good to me.

> We might want to add another ConfigMap anyways (for adding upcoming multi TUF root work), so that might be something. 

👍🏻 

> I think having an option that says if an image does NOT match any policy should be denied should be there.

Why would we deny any image that doesn't match the policy ? If it doesn't match the globbing, then it should be excluded of the restrictions.